pkg/agent/flowexporter/connections/connections_windows.go

```diff
@@ -0,0 +1,17 @@
+// +build windows
```

Comment:
Do we need _windows.go for all these files? Could limit the funcs referenced outside in a single file, and so we just need a single _windows.go file?

Reply:
I moved everything into one file. Having interface in  common file and defining functions in corresponding linux/windows because of dependency on linux specific conntrack library.